integration-tests/tests/test_integration.py

```diff
@@ -430,24 +444,27 @@ def agent_hotload(self, sync_accounts, agent_keystore, agent_keystore_path, tmp_
             yield
 
     @pytest_asyncio.fixture
-    async def client(self, agent):
+    async def client(self, agent, tmp_path):
         client = PythAgentClient(address="ws://localhost:8910")
         await client.connect()
-        yield client
+        yield (client, tmp_path)
         await client.close()
 
     @pytest_asyncio.fixture
     async def client_hotload(self, agent_hotload):
         client = PythAgentClient(address="ws://localhost:8910")
         await client.connect()
-        yield client
+        yield (client, tmp_path)
```

Comment:
couldn't we add another fixture for the path specifically? It doesn't look very clean like this 🤔

Reply:
oh, you're right, I should just mention the fixture directly in test signature, thanks!